### DISCUSSION TITLE
Custom Order Fields not visible in Order Confirmation

### DISCUSSION THREAD

**Discussion by leonkosterss** [swellstores/community#539] [Help]
Hello,

I added custom fields to an Order and to the Order confirmation.

When I create an Order by the dashboard, it sends the mail with the new custom fields.

But when I send it with a Custom Checkout, it adds the fields to the Order, but not in the Order Confirmation.

The webshop is new.obimex.nl

Any reason why this is happening?

**Comment by logeshswell** [comment]
> But when I send it with a Custom Checkout

1. What do you send from your custom checkout?
2. Do you use swell-js or swell-node in your custom checkout?

**Reply by logeshswell** [reply]
Can you share some additional information with screenshots so I can understand the issue better?
